src/webportal/src/app/job-submission/components/command-section.jsx

```diff
@@ -43,8 +43,9 @@ export const CommandSection = (props) => {
     <BasicSection sectionLabel='Command'>
       <FormShortSection>
         <MonacoTextFiled
-          monacoProps={{height: 200}}
+          monacoProps={{height: 250, language: 'shell'}}
```

Comment:
Need to add 'shell' to this line?
[monaco-editor.jsx#L110](https://github.com/microsoft/pai/blob/3fe5ad0c909f8d2b037a25311741a1e12c450a80/src/webportal/src/app/components/monaco-editor.jsx#L110)

Reply:
thx, i'll fix this.